_monitoring-plugins/alerting/monitors.md

```diff
@@ -196,6 +193,7 @@ Steps to create a trigger differ depending on whether you chose **Visual editor*
 
 You begin by specifying a name and severity level for the trigger. Severity levels help you manage alerts. A trigger with a high severity level (e.g. 1) might page a specific individual, whereas a trigger with a low severity level might message a chat room.
 
+Remember that per-query monitors run your trigger's script just once against the query's results, but per-bucket monitors execute your trigger's script on each bucket, so you should create a trigger that best fits the monitor you chose. If you want to execute multiple scripts, you must create multiple triggers.
 
 ### Visual editor
```

Comment:
The "Visual editor" and "Extraction query" sections for `Create triggers` below are describing the query-level Monitor behavior, did we want to update that to differentiate from bucket-level Monitors?